command/operator_debug.go

```diff
@@ -423,6 +429,20 @@ func (c *OperatorDebugCommand) Run(args []string) int {
 	}
 	c.index = uint64(eventIndex)
 
+	// Parse the pprof capture interval
+	pi, err := time.ParseDuration(pprofInterval)
+	if err != nil {
+		c.Ui.Error(fmt.Sprintf("Error parsing interval: %s: %s", pprofInterval, err.Error()))
```

Comment:
Ideally the error message should match the flag name exactly, but especially here because there's already an `-interval` flag:

```suggestion
		c.Ui.Error(fmt.Sprintf("Error parsing pprof-interval: %s: %s", pprofInterval, err.Error()))
```